frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -86,6 +86,11 @@ const footerVariantStyles = {
   `,
 };
 
+export const ParametersWidgetContainer = styled.div`
```

Comment:
the parameter list in all embeddings (including public sharing) uses a different container, so this style is to make it renders like `ParametersWidgetContainer` inside `frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx`